src/main/resources/assets/js/app/settings/wizard/ProjectItemNameWizardStepForm.ts

```diff
@@ -63,6 +64,12 @@ export class ProjectItemNameWizardStepForm
         });
     }
 
+    public validate(_silent?: boolean): ValidationRecording {
```

Comment:
Not directly related to this PR, but from what I see parameter `_silent` is never used (correct me if I'm wrong). So we can remove it from here and its parent `WizardStepFrom` class.